test/integration/scheduler/queueing/queue.go

```diff
@@ -433,6 +436,16 @@ var CoreResourceEnqueueTestCases = []*CoreResourceEnqueueTestCase{
 			st.MakeNode().Name("fake-node1").Capacity(map[v1.ResourceName]string{v1.ResourceCPU: "4"}).Obj(),
 			st.MakeNode().Name("fake-node2").Capacity(map[v1.ResourceName]string{v1.ResourceCPU: "2"}).Label("group", "b").Obj(),
 		},
+		InitialDeviceClasses: []*resourceapi.DeviceClass{
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "gpu-device-class",
+				},
+				Spec: resourceapi.DeviceClassSpec{
+					ExtendedResourceName: ptr.To("example.com/gpu"),
+				},
+			},
+		},
```

Comment:
why do you need this InitialDeviceClasses ? it was not needed before.

Reply:
In testExtendedResource() before the test starts, you created a device class

	c, err := tCtx.Client().ResourceV1().DeviceClasses().Create(tCtx, testClass, metav1.CreateOptions{})

this device class should suffice? 

is it due to the race between the test, and informer update that causes: The global cache has no mapping for "example.com/gpu" ? what if you add a deley after the class creation, before the test start with something like:

time.Sleep(10*time.Seconds)

Reply:
The `testExtendedResource()` function and `TestCoreResourceEnqueue` are in completely different test files:

  1. `testExtendedResource()`: In `test/integration/dra/dra_test.go`
  2. `TestCoreResourceEnqueue`: In `test/integration/scheduler/queueing/queue.go`

I think, each test suite creates its own isolated environment:
  So the `DeviceClass` created in `testExtendedResource()` is NOT visible to `TestCoreResourceEnqueue`

Reply:
I see. that makes sense.